elements/order_slip.php

```diff
@@ -123,11 +123,11 @@
         </p>
 
         <p>
-            <strong><?= t("Payment Method") ?>: </strong><?= $order->getPaymentMethodName() ?><br>
+            <strong><?= t("Payment Method") ?>: </strong><?= t($order->getPaymentMethodName()) ?><br>
```

Comment:
Ok, please give me a hint where to find that config item. I haven't found s.th. like that in the package. thx (maybe i'm a bit blind ;) )